docs/source/coworkers.rst

```diff
@@ -0,0 +1,43 @@
+Meeting your Coworkers
+======================
+
+Send a Slack Message to the Team
+--------------------------------
+Introduce yourself to the team with a Slack Message to the tcril-team channel.
+Share the amount of personal information you are comfortable with.
+And feel free to ask an ice-breaker question at the end to engage your coworkers.
+
+Here's an example:
+
+ | Hello @channel!
+ | My name is Bob Smith, and I'm excited to join tCRIL.
+ | I am originally from New York, NY, USA and currently live in Cambridge, MA, USA.
+ | I've worked 5 years as a Software Engineer.
+ | My favorite type of food is Pizza.
+ | My favorite movie is Back to the Future.
+ | What's everyone's favorite animal?
+
+Meet with Manager to review 30/60/90
+------------------------------------
+Your first couple of days at tCRIL are very important to ensuring the type of success you deserve to achieve.
+Meeting with your manager to discuss a plan that will guide your first 30 days, 60 days, 90 days, and beyond will start you off on the right foot.
+
+Set up Social 1-on-1’s
+----------------------
+While the Open edX community is very large, tCRIL is a smaller group.
+Take some time out of your schedule to set up Social 1-on-1's to get to know your coworkers better.
+
+Check out the Organization Chart
+--------------------------------
+While tCRIL is a smaller organization, there may still be some confusion on who to ask questions to.
+Familiarize yourself with the Organization Chart and it will be easier to ask questions to the right person when the time comes.
+
+Participate in Standups
+-----------------------
+Every morning, the Engineering team gathers together in a short meeting called a Standup.
+Use this meeting to sync up with the rest of your team on progress being made on project tasks.
+
```

Comment:
```suggestion
Non-Engineering team members are welcome to drop in on standup meetings, otherwise check with your manager on team-specific meetings relevant to you.

```

Reply:
Added